docs/sources/alerting/fundamentals/notifications/index.md

```diff
@@ -48,8 +65,18 @@ The following table lists the contact point types supported by Grafana.
 | [WeCom](#wecom)                                  | `wecom`                   | Supported            | N/A                                                                                                      |
 | [Zenduty](https://www.zenduty.com/)              | `webhook`                 | Supported            | N/A                                                                                                      |
 
-## Useful links
+## Templating notifications
+
+In Grafana it is possible to customize notifications with templates. For example, templates can be used to change the subject and message of an email, or the title and message of notifications sent to Slack.
+
+A template is not limited to an individual integration or contact point either, but instead can be used in a number of integrations in the same contact point and even integrations across different contact points. For example, a Grafana user can create a template called `custom_subject_or_title` and use it for both templating subjects in emails and titles of Slack messages without having to create two separate templates.
```

Comment:
This is where I think the phrasing used in the UI is not helpful. Somehow we're using the word "contact point type" to mean a configured integration in a contact point, when I think it really should mean the type of integration (i.e. email). You can also have multiple contact point "types" in the same contact point. It's strange when read like that.

However, what I am trying to explain here is that not only can you use the same template in different contact point types (i.e. email and slack), you can also use them in a number of similar "types" in the same contact point, and in different contact points.

I'm considering if we should change it from contact point type to integration because I think the current name is confusing.

Reply:
I struggled on the wording here too, this took me like 3 passes and I agree it's not perfect 😀

I was mainly just trying to shorten it a bit, the original wording felt a little too drawn out, but it's really a minor nit.